packages/office-ui-fabric-react/src/components/FocusZone/FocusZone.tsx

```diff
@@ -232,6 +250,112 @@ export class FocusZone extends BaseComponent<IFocusZoneProps, {}> implements IFo
     }
   };
 
+  /**
+   * Callback hooked to the root element. When we receive a blur, we want
+   * to detect if the blur is due to an element being removed from the Zone.
+   * This is done by evaluating if the destination (in ev.relatedTarget) is
+   * null. If so, we need to move the focus to either the next element, or
+   * possibly, the previous element if there is no "next".
+   *
+   * In the case there are no more focusable elements in the zone, we "park"
+   * focus on the root temporarily. If the user tabs elsewhere, we remove
+   * focusablilty from the root. After a zone re-render, if focus is still
+   * "parked", we try to move focus back to the first focusable child.
+   *
+   * Note: Attempting to move this to a React event, rather than using a
+   * native event, broke the implementation.
+   */
+  private _onBlurCapture(ev: FocusEvent) {
+    const { current: root } = this._root;
+    const { target: from } = ev;
+    const isFocusDisappearing = ev.relatedTarget === null || ev.relatedTarget === document;
```

Comment:
Does this work in IE11? I know [this linksays that `relatedTarget`](https://developer.mozilla.org/en-US/docs/Web/API/FocusEvent/relatedTarget) should be available, but I've run into instances where it's not even when this site says it is.

Reply:
this was removed.